### ISSUE TITLE
jscad cli does not work via clean install on fresh linux machine

### ISSUE THREAD

**Issue by DavidLyon66** [jscad/OpenJSCAD.org#1342]
## Expected Behavior
Jscad CLI to be able to run from command line after installation on linux

## Actual Behavior
Jscad cli interface does not work with modeling

## Steps to Reproduce the Problem

  1. install jscad cli as per documented instructions on a fresh ubuntu installation ie, sudo npm install -g @jscad/cli
  2. save a jscad example from off openjscad.xyz to a disk file
  3. run 'jscad <sourcefile> -o <destfile>, an error is shown and the executable will not work.

## Specifications

  - Version: 2.12
  - Platform: linux ubuntu
  - Environment: (browser, local server, node.js etc) bash

![2024-04-24-213431](https://github.com/jscad/OpenJSCAD.org/assets/122891826/83a6d980-8f57-4b23-a5fd-3abd0773b74c)
![2024-05-03-033421](https://github.com/jscad/OpenJSCAD.org/assets/122891826/ea312da3-55c1-4917-8178-76dc2464e8d0)
![2024-05-03-035320](https://github.com/jscad/OpenJSCAD.org/assets/122891826/702db42e-2491-4198-9f92-2ee146d76f05)
![2024-05-03-040155](https://github.com/jscad/OpenJSCAD.org/assets/122891826/2e43f86f-f26e-4fac-88dc-330891890640)

**Comment by z3dev** [comment]
@DavidLyon66 Yeah. You're correct. The 'global' installation does not work, and the documentation needs to be corrected.

The proper way to use the CLI is via a local project. There are many reasons for this but here's a really good discussion about the general issue of installing and maintaining global NPM packages. It's not a good idea.

https://www.sitepoint.com/solve-global-npm-module-dependency-problem/

I suggest creating a new local NPM package, and install in the CLI as a dependency. You can then maintain that local package (and dependencies), and run the CLI in scripts, etc.

And if you really really need a global CLI then use NPM link.
